setup.py

```diff
@@ -37,6 +37,7 @@ def read(fname):
             "autometa-binning = autometa.binning.recursive_dbscan:main",
             "autometa-unclustered-recruitment = autometa.binning.unclustered_recruitment:main",
             "autometa-binning-summary = autometa.binning.summary:main",
+            "autometa-download-dataset = autometa.validation.datasets:main",
```

Comment:
You need to add this endpoint to the end of the `Dockerfile`, where you have a test of the other endpoints.